README_V8.md

```diff
@@ -0,0 +1,549 @@
+
+### Rate Limiter
+
+<!-- snippet: rate-limiter -->
+```cs
+// Create a rate limiter with default options that limit to 1000 concurrent executions and no queue.
+new ResiliencePipelineBuilder()
+    .AddRateLimiter(new RateLimiterStrategyOptions());
+
+// Create a rate limiter to allow a maximum of 100 concurrent executions and a queue of 50.
+new ResiliencePipelineBuilder()
+    .AddConcurrencyLimiter(100, 50);
+
+// Create a rate limiter that allows 100 executions per minute.
+new ResiliencePipelineBuilder()
+    .AddRateLimiter(new SlidingWindowRateLimiter(new SlidingWindowRateLimiterOptions
+    {
+        PermitLimit = 100,
+        Window = TimeSpan.FromMinutes(1)
+    }));
+
+// Create a custom partitioned rate limiter.
+var partitionedLimiter = PartitionedRateLimiter.Create<Polly.ResilienceContext, string>(context =>
+{
+    // Extract the partition key.
+    string partitionKey = GetPartitionKey(context);
+
+    return RateLimitPartition.GetConcurrencyLimiter(
+        partitionKey,
+        key => new ConcurrencyLimiterOptions
+        {
+            PermitLimit = 100
+        });
+});
+
+new ResiliencePipelineBuilder()
+    .AddRateLimiter(new RateLimiterStrategyOptions
+    {
+        // Provide a custom rate limiter delegate.
+        RateLimiter = args =>
+        {
+            return partitionedLimiter.AcquireAsync(args.Context, 1, args.Context.CancellationToken);
+        }
+    });
+```
+<!-- endSnippet -->
+
+Example execution:
+
+<!-- snippet: rate-limiter-execution -->
+```cs
+var pipeline = new ResiliencePipelineBuilder().AddConcurrencyLimiter(100, 50).Build();
+
+try
+{
+    // Execute an asynchronous text search operation.
+    var result = await pipeline.ExecuteAsync(
+        token => TextSearchAsync(query, token),
+        cancellationToken);
+}
+catch (RateLimiterRejectedException ex)
+{
+    // Handle RateLimiterRejectedException,
+    // that can optionally contain information about when to retry.
+    if (ex.RetryAfter is TimeSpan retryAfter)
+    {
+        Console.WriteLine($"Retry After: {retryAfter}");
+    }
+}
+```
+<!-- endSnippet -->
+
+Rate limiter strategy throws `RateLimiterRejectedException` if execution is rejected. For more details see [Rate Limiter strategy documentation](https://github.com/App-vNext/Polly/wiki/Rate-Limit).
+
+## Next steps
+
+To learn more about Polly, visit the [documentation](docs/README.md) or check out the [samples](#samples).
+
+## Release notes
+
+- The [changelog](https://github.com/App-vNext/Polly/blob/main/CHANGELOG.md) describes changes by release.
+- We tag Pull Requests and Issues with [milestones](https://github.com/App-vNext/Polly/milestones) which match to NuGet package release numbers.
+- Breaking changes are called out in the wiki ([v7](https://github.com/App-vNext/Polly/wiki/Polly-v7-breaking-changes); [v6](https://github.com/App-vNext/Polly/wiki/Polly-v6-breaking-changes)) with simple notes on any necessary steps to upgrade.
```

Comment:
I guess this is redundant for v8, as we'll have a separate breaking changes document not in the Wiki.

Reply:
Dropped